_data/tobacco.yml

```diff
@@ -76,6 +76,18 @@ websites:
       othercrypto: yes
       doc: https://www.reddit.com/r/btc/comments/7jq1hc/we_just_added_bitcoin_cash_as_a_payment_method_on/
 
+    - name: Liquido24
+      url: https://www.liquido24.de/
```

Comment:
Can you add the `lang` Tag for these .de sites if they’re primarily for that language. Also, if they’re regional add the region tag and country tags/city tags where pissible